src/common/js/timesheet/index.js

```diff
@@ -0,0 +1,244 @@
+import $ from 'jquery'
+import styles from './index.scss'
+
+// 残業申請ページ表示に要するパラメータ
+const overtimeReportState = {
+  status: 'default',
+  module: 'application_form',
+  action: 'editor',
+  application_form_master_id: '122'
+}
+// 各種申請画面
+const appFormState = {
+  module: 'application_form',
+  action: 'application_form'
+}
+
+// 残業申請画面
+const overtimeReportPath = __DEV__
+  ? 'overtime_report.html'
+  : '/'
+
+// 各種申請画面
+const appFormPath = __DEV__
+  ? 'application_form.html'
+  : `/?${$.param(appFormState)}`
+
+/**
+ * 初期実行処理
+ *
+ * 各種申請画面から残業申請フォームIDを取得したのち、行追加処理を開始
+ */
+export default () => {
+  $.ajax({
+    url: appFormPath,
+    dataType: 'html'
+  })
+  .done(function(data) {
+    overtimeReportState.application_form_master_id = $(data).find('#slct_appformmasterid').eq(0).find('option').filter(function(){
+      return $(this).text() === '残業申請*';
+    }).val();
+    addRow();
+    if(__BOOKMARKLET__){
+      alert('残業申請ボタン列を追加しました。\n\nボタンを押して残業申請画面に進み、\n再度ブックマークレットを実行してください。');
```

Comment:
Please replace with the following wording of the master branch.
https://github.com/umesan/overtime-application/blob/master/src/common/js/export.js#L39